nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -323,3 +323,5 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 - QtMultimedia has changed its default backend to `QT_MEDIA_BACKEND=ffmpeg` (previously `gstreamer` on Linux or `darwin` on MacOS).
   The previous native backends remain available but are now minimally maintained. Refer to [upstream documentation](https://doc.qt.io/qt-6/qtmultimedia-index.html#ffmpeg-as-the-default-backend) for further details about each platform.
 
+- New module `hardware.display` streamlines working with custom display configurations allowing workarounds for misbehaving monitors
```

Comment:
Please see the comment under the next headline

Reply:
You mean I should not put it at the end, but somewhere in the middle of the list?